src/js/components/ErrorMessage.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { PropTypes } from 'react';
+
+export function ErrorMessage(props) {
+    const extraClass = props.className;
+    return (
+        <div className={`ErrorMessage ${extraClass}`}>{props.children || props.text}</div>
```

Comment:
Is there a reason we need the ability to specify `<ErrorMessage text="foo"/>` instead of just `<ErrorMessage>foo</ErrorMessage>`?